app/src/main/java/fr/gaulupeau/apps/Poche/data/StorageHelper.java

```diff
@@ -114,4 +118,24 @@ public static boolean deleteFile(String path) {
         return new File(path).delete();
     }
 
+    public static File dumpQueueData(String data) throws IOException {
+        if (!isExternalStorageWritable()) {
+            throw new IllegalStateException("External storage is not writable!");
+        }
+
+        String path = getExternalStoragePath() + "/"
+                + "Local_changes_"
+                + new SimpleDateFormat("yyyyMMdd_HHmmss").format(new Date())
+                + ".txt";
+        File file = new File(path);
+
+        file.createNewFile();
```

Comment:
We should check the return value here. It's false if the file already exists. This could accidentally happen if i touch two times in a second.

Reply:
I don't see any point in checking for the result *in this particular ad hoc solution*, since the file should exist after that call either way (and it will be simply overwritten if you call it twice in a second). I added a comment.

I considered adding milliseconds to file name, but I don't think it's important.